docs/peopleops/offboarding/offboarding-process.md

```diff
@@ -0,0 +1,51 @@
+# Offboarding process
+
+Thoth Tech understands the importance of handover and offboarding. Reasons include:
+
+- data security and risk management;
+- efficient organisation and company administration;
+- a chance to get final feedback and review from exiting members; and
+- a opportunity to place the company in a strong position to continue developing projects that run
+  across capstone trimesters.
+
+## Update and correctly store all relevant documents
+
+Ensure that all projects' documentation and processes are up-to-date and have been stored in the
+correct location of the relevant [Thoth Tech repository](https://github.com/thoth-tech) on GitHub:
+
+- Documentation to [Documentation](https://github.com/thoth-tech/documentation) repository
+- Processes to [Handbook](https://github.com/thoth-tech/documentation)
+- Code to relevant project repository
+
+Documentation should be clear enough for subsequent members to easily follow, understand and, as
+necessary, continue work on projects.
+
+## Handover Document
+
+Complete handover document and showcase according to the capstone requirements for the relevant
+trimester. Ensure ALL relevant PR or document links are included. Ensure a markdown copy of the
+handover document is uploaded to the relevant
+[Thoth Tech repository](<(https://github.com/thoth-tech)>)
+
+## Exit Survey
+
+Exiting seniors should undertake the
+[Senior Exit Interview](https://forms.office.com/Pages/DesignPage.aspx?origin=NeoPortalPage#FormId=7Hgj0IgW1UaFQBwotfRw9mRQ8baZZ01Ep6tAGY0fXmNUNDZUSVM3R0UzRDc3UDgxQkdCVVIzVUNHWS4u)
+survey so that feedback can be collected and assessed to allow the Thoth Tech leadership to see what
+went well and what opportunities for improvement may exist.
+
+Juniors should complete the
+[What do you want to do next?](https://forms.office.com/Pages/DesignPage.aspx?origin=NeoPortalPage#FormId=7Hgj0IgW1UaFQBwotfRw9pCshblNu_9FhxlIpGGIxEhUQ1RHNk5IWEs1SVlBSE1QOUxNMERQUFFTUy4u)
+form to give the leadership team (current and incoming) an opportunity to reassess the member's role
+and to plan for projects and team and role allocation for the next trimester.
+
+## Remove Accounts & Access
+
+For all exiting members:
+
+- [ ] Remove Thoth Tech [GitHub](https://github.com/orgs/thoth-tech/teams) access
+- [ ] Remove from Thoth Tech [Trello](https://trello.com) boards
+- [ ] Remove from company Teams Channels
+- [ ] Cancel/remove any recurring team meetings
+- [ ] Remove Thoth Tech [Miro](https://miro.com) access
+- [ ] Remove any Thoth Tech [Form](https://forms.office.com/) access
```

Comment:
I think this would be removed automatically as part of the team channels access removal 🤔 The team member will no longer have access the the Sharepoint folders and associated docs, forms included.

Reply:
Ah, cool - just tagged you on something similar @tancnle. You answered half my question (sorry, I copied these straight from the categories in the ownership Trello card). Not sure if I should mention at all; as mentioned previously, may be an issue for those who created their own teams (maybe) but they probably shouldn't have done that in the first place, so.... I might just remove.